src/main/java/nl/tudelft/contextproject/camera/LiveCameraConnection.java

```diff
@@ -140,33 +153,37 @@ private String sendRequest(URL url) throws IOException {
             connected = false;
             return "";
         }
+
         BufferedReader reader = new BufferedReader(new InputStreamReader(connection.getInputStream(), "UTF-8"));
         String response = reader.readLine();
         reader.close();
+
         return response != null ? response : "";
     }
-    
+
     /**
      * Returns true if the camera is on auto focus.
-     * @return true if the camera is on auto focus.
+     * @return True if the camera is on auto focus.
      */
     public boolean hasAutoFocus() {
         if (connected) {
             try {
                 String autoFocusRes = sendRequest(buildPanTiltHeadControlURL("%23D1"));
+
                 if (autoFocusRes.startsWith("d1")) {
                     autoFocus = Integer.parseInt(autoFocusRes.substring(2)) == 1;
                 } else {
-                    throw new IOException("Wrong response from camera: " + autoFocusRes);
+                    throw new IOException(errorString + autoFocusRes);
                 }
             } catch (IOException e) {
                 e.printStackTrace();
                 return autoFocus;
             }
         }
+
         return autoFocus;
     }
-    
+
     /**
      * Sets the auto focus setting on the camera to on (true) or off (false).
      * @param autoFocus true for ON, false for OFF
```

Comment:
Shouldn't you insert a whitline here because in all other javadocs there is a whiteline between the description and the parameter.